source/Mlos.Python/mlos/Optimizers/ExperimentDesigner/UtilityFunctionOptimizers/GlowWormSwarmOptimizer.py

```diff
@@ -77,7 +77,7 @@ def __init__(
         self.dimension_names = [dimension.name for dimension in self.parameter_adapter.dimensions]
 
     @trace()
-    def suggest(self, context_values_dataframe=None):  # pylint: disable=unused-argument
+    def maximize(self, target_function, context_values_dataframe=None):  # pylint: disable=unused-argument

```

Comment:
What's the benefit of doing this?

Reply:
we can optimize any function, not just the acquisition function. I'm now a bit less certain that we really need that. I thought directly maximizing the surrogate instead of the acquisition function would be more useful for the optimum.